utils/spacewalk-common-channels.ini

```diff
@@ -2164,3 +2164,58 @@ repo_type = deb
 checksum = sha256
 base_channels = astralinux-orel-pool-amd64
 repo_url = https://download.astralinux.ru/astra/stable/orel/repository/dists/orel/non-free/binary-amd64/
+
+[almalinux8]
+archs    = x86_64
+name     = AlmaLinux 8 (%(arch)s)
+gpgkey_url = https://repo.almalinux.org/almalinux/RPM-GPG-KEY-AlmaLinux
```

Comment:
I *really* want to fix the reposync issue before marking this as Stable. We should really be using mirrors.

Reply:
Ok, as agreed with  @paususe, let's merge it with the reposync problem fixed.

However we need to remember: we'll need a SQL migration script when we fix the problem. I will add that to the issue.